### PR TITLE
Add PHP and WP requires headers

### LINKED ISSUE #198: Add PHP and WP requires headers
Ensure the requires headers in the plugin's base PHP file and readme match our given versions on the RBP website.

### PR DESCRIPTION
**Pull request by joelyoder**
Fixes #198

### CODE REVIEW

File: client-dash.php [hunk 1]
```diff
@@ -9,6 +9,8 @@
  * Plugin Name: Client Dash
  * Description: Creating a more intuitive admin interface for clients.
  * Version: 2.1.6
+ * Requires at least: 4.7.0
```
Comment (d4mation) [inline client-dash.php:12]: 4.7 seems suspiciously high, but that _is_ what's in our `readme.txt` 🤔 

Would you be able to run the ZIP from the .Org directory through https://wpseek.com/pluginfilecheck/ to double-check that for me?

**Comment by joelyoder** [comment]
> The plugin 'Client Dash' is suggested to use with WordPress 4.8.0 and above for best experience and all features working.

Seems to be because it uses `wp_enqueue_editor()` which was introduced in 4.8.

**Comment by joelyoder** [comment]
It also requires 4.7 since it's using `register_rest_field()`

**Comment by joelyoder** [comment]
The equeue editor function is actually in RBM FH

**Comment by d4mation** [comment]
> `wp_enqueue_editor()` in RBM FH

Ah, yep. That's definitely a big one. I'm not sure if this plugin technically every calls the functions within RBM FH that would cause it to run that, but RBM FH itself does rely on 4.8 for that reason.

**Review by d4mation — APPROVED** [review]

**Pull request merged**